pkg/logqlmodel/stats/context.go

```diff
@@ -172,7 +172,7 @@ func (r *Result) Merge(m Result) {
 	r.Querier.Merge(m.Querier)
 	r.Ingester.Merge(m.Ingester)
 	r.ComputeSummary(time.Duration(int64((r.Summary.ExecTime+m.Summary.ExecTime)*float64(time.Second))),
-		time.Duration(r.Summary.QueueTime+m.Summary.QueueTime))
+		time.Duration(int64((r.Summary.QueueTime+m.Summary.QueueTime)*float64(time.Second))))
```

Comment:
What do you think about creating a helper function for this since, it seems we're performing the same operation many times in this file and others?

Reply:
Good point 👍 
Added in commit https://github.com/grafana/loki/pull/5052/commits/d90902a75c9a58c24e2570ed44581b204453f399

Reply:
Looks great, thanks!